server/bot/commands/utilities.ts

```diff
@@ -518,3 +518,48 @@ export const handleEditLastSay: Command = {
     } catch (e) { return 'Nothing to edit.' }
   }
 }
+
+export const handleSuppress: Command = {
+  name: 'suppress',
+  opts: {
+    requirements: {
+      permissions: { manageMessages: true }
+    },
+    description: 'Suppress or unsuppress embeds in a message.',
+    fullDescription: 'Suppress embeds in a message. If the message already has a suppressed embed, it will get unsuppressed. Requires \'Manage Messages\' permission',
+    usage: '/suppress (channel) <message ID or link>',
+    example: '/suppress #general 123456789012345678'
+  },
+  generator: async (message, args, { client }) => {
+    let msg
+    let channelToEdit
```

Comment:
Rename the variable to contain a channel itself.
```suggestion
    let channel
```